org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/info/InfoWindow.java

```diff
@@ -913,7 +913,7 @@ protected ColumnInfo createLookupColumnInfo(TableInfo[] tableInfos,
 		if (! colSQL.toUpperCase().contains(" AS "))
 			colSQL += " AS " + infoColumn.getColumnName();
         editorMap.put(colSQL, editor);
-        Class<?> colClass = columnName.endsWith("_ID") ? KeyNamePair.class : String.class;
+        Class<?> colClass = columnName.endsWith("_ID") || columnName.endsWith("tedBy") ? KeyNamePair.class : String.class;
```

Comment:
Safer to just check equal for CreatedBy and UpdatedBy. You never know whether this will break anything out there.

Reply:
Commited Changes. Changed to CreatedBy and UpdatedBy